dlt/common/schema/typing.py

```diff
@@ -276,14 +276,43 @@ class TScd2StrategyDict(TMergeDispositionDict, total=False):
 ]
 
 
+TReferenceCardinality = Literal["-", "<", ">", "<>"]
+"""Represents cardinality between `column` (left) and `referenced_column` (right)
+`-`: one-to-one
+`<`: one-to-many
+`>`: many-to-one
+`<>`: many-to-many
+
+Note that `column <> referenced_column` is equivalent to specifying
+both `column < referenced_column` and `column > referenced_column` 
+"""
+
+
 class TTableReference(TypedDict):
     """Describes a reference to another table's columns.
     `columns` corresponds to the `referenced_columns` in the referenced table and their order should match.
     """
 
+    label: NotRequired[str]
+    """Label to describe the relation 'liked'."""
```

Comment:
I do not understand this comment :)  is it a custom annotation? you can always add as many `x-annotation-` as you want.

Reply:
Yeah, it is called "label" or "edge label" in graph databases. I'd just improve the comment so it is clear what is meant by this.